app/views/subscription_mailer/send_digest.html.erb

```diff
@@ -20,7 +20,8 @@
           </div>
           <div style="float: left;">
             <b style="font-weight: 500;"><%= n.author.username.capitalize %></b>
-            <p style="margin: 0;color: #999;font-weight: 500;">Published Feb 2</p>
+            <p style="margin: 0;color: #999;font-weight: 500;">Published <%= Time.now.strftime("%m/%d") %>
```

Comment:
Could you please explain what this will do? Will this show the date at the time the mail has been sent?

Reply:
@IshaGupta18  actually everytime the mail that was being sent , it used to have a particular date of feb 2 . But we have to put the current date . Thats why

Reply:
Secondly I haven't made any changes to the gemfile

Reply:
Then please see why the travis checks are failing and kindly rectify them.

Reply:
@IshaGupta18  yeah sure ..... can u please tell which ll parts have u solved so that I can work on others